src/views/notifications/components/addedModeratorNotification.js

```diff
@@ -0,0 +1,47 @@
+// @flow
+import * as React from 'react';
+import { parseEvent, parseNotificationDate, parseContext } from '../utils';
+import { ActorsRow } from './actorsRow';
+import { NotificationCard, TextContent, JoinContext, Content } from '../style';
+import Icon from '../../../components/icons';
+import {
+  CardLink,
+  CardContent,
+} from '../../../components/threadFeedCard/style';
+import markSingleNotificationSeenMutation from 'shared/graphql/mutations/notification/markSingleNotificationSeen';
```

Comment:
not sure how or when i should use this?
I noticed some `MiniNotification` variations for other Notifications. Do we need it here as well? Dont exactly know what they are for.

Reply:
`MiniNotification` is for the dropdown, the other one is for viewing `/notifications` :)